docs/medical-api/api-reference/fhir/consolidated-data-query-get-single.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Get Single Consolidated Data Query status"
```

Comment:
maybe "Get Specific" to be consistent with the below?

Reply:
opted to just change the language below - wanted to highlight that this query returns a single entity instead of a list. Usually i do that with "Get Entity" and "Get Entities" named requests, but didn't want to go renaming a bunch of stuff on a public api LOL